app/views/rsvp.erb

```diff
@@ -50,6 +50,14 @@
     </div>
   </div>
 
+ <!-- Emaill Id - TextBox -->
+ <div class="form-group">
+    <label class="col-md-4 control-label"for="guestEmail">Email address please:</label>
+    <div class="col-md-4">
+    	<input type="email" name="guestEmail" class="form-control" id="guestEmail" placeholder="Enter email">
```

Comment:
Nope, this isn't what we want.
Facebook already gives us a users email, persist that in the background.